frontend/src/app/features/quiz/result/result-character/result-character.component.ts

```diff
@@ -0,0 +1,161 @@
+        this.characterTitle =
+          '“เราจะเป็นมิตรกับคนทั้งกาแลคซี่ และเป็นความแฮปปี้ให้เธอด้วย”';
+        this.characterImgLink = 'images/character-peacy.png';
+        this.characterDescription =
+          'แสงของเธอทำให้สังคม…รวมเป็นหนึ่งเดียว เป็นผู้ไกล่เกลี่ยชั้นเยี่ยม เห็นใจ รู้ใจ เข้าใจ เสมือนว่าเป็นเขา ทำให้ทุกคนยอมรับ เป็นผู้ให้ที่สังคมขาดไม่ได้เลยทีเดียว';
+        this.characterAttributes = [
+          'เป็นมิตร ร่าเริง เป็นกันเอง',
+          'ใจกว้าง โอบอ้อมอารี',
+          'เป็นผู้ฟังที่ดี เข้าใจผู้อื่น',
+          'มองโลกในแง่ดี เสียสละ',
+          'นักสร้างความสัมพันธ์ เอาใจใส่คนรอบข้าง',
+        ];
+        this.attributesChipColor = 'bg-[#ffd76b]';
+        this.shinningMethod =
+          '“ส่องแสงของเธอด้วยการส่งต่อความสดใสให้คนรอบข้าง! เริ่มจากส่งข้อความให้กำลังใจไปยังเพื่อนสนิทให้เขาช็อคไปเลยเป็นไง? ٩(◕‿◕｡)۶ ”';
+        break;
+      case Character.DREAMY:
+        this.bgLink = 'images/orange-bg-full.png';
+        this.characterNameEn = 'Dreamy';
+        this.characterNameTh = 'ผู้สุนทรีย์';
+        this.characterTitle = '“ทุกสิ่งที่เธอฝัน เราจะช่วยทำมันให้เป็นจริง”';
+        this.characterImgLink = 'images/character-dreamy.png';
+        this.characterDescription =
+          'แสงของเธอทำให้สังคม…เชื่อว่าทุกฝันเกิดขึ้นได้จริง ทุกปัญหาจะแก้ไขได้ เธอคลายทุกความกังวล และเปลี่ยนเป็นความสดใสพร้อมกับสิ่งใหม่ที่กำลังจะเกิดขึ้น';
+        this.characterAttributes = [
+          'โดดเด่น มีเสน่ห์',
+          'รักอิสระ มองโลกในแง่ดี',
+          'สร้างสรรค์ ช่างจินตนาการ',
+          'มีมนุษยสัมพันธ์ เข้ากับคนง่าย',
+          'แก้ไขปัญหาเฉพาะหน้าได้ดี กระตือรือร้น',
+        ];
+        this.attributesChipColor = 'bg-[#ef8923]';
+        this.shinningMethod =
+          '“ส่องแสงของเธอด้วยการปรับนิดเปลี่ยนหน่อยสิ่งของที่เธอมีให้กลายเป็นไอเดียใหม่ที่น่าสนใจ! เอาเสื้อผ้าในตู้มา match ใหม่ ให้กลายเป็น new look ดูสิ! (๑˃ᴗ˂)ﻭ ”';
+        break;
+      case Character.MEKKA:
+        this.bgLink = 'images/green-bg-full.png';
+        this.characterNameEn = 'Mekka';
+        this.characterNameTh = 'ผู้จริงใจ';
+        this.characterTitle =
+          '“คนน่ารักที่เธอบอกว่าน่ากอด เค้าซัพพอร์ตเธอได้เท่าเรามั้ย”';
+        this.characterImgLink = 'images/character-mekka.png';
+        this.characterDescription =
+          'แสงของเธอทำให้สังคม…ไม่ขาดแคลนคนทำจริง มีความหวัง รู้สึกปลอดภัยในทุกการกระทำเพราะเธอจะคอยรับฟัง เป็นที่พึ่ง และซัพพอร์ตทุกคนด้วยความเข้าใจเสมอ';
+        this.characterAttributes = [
+          'นักสนับสนุน และช่วยเหลือทุกคน',
+          'สุภาพ อ่อนน้อมถ่อมตน',
+          'มีความประนีประนอม',
+          'อดทน รับฟังคนอื่นได้ดี',
+          'รักสงบ ชื่นชอบความเรียบง่าย',
+        ];
+        this.attributesChipColor = 'bg-[#50eac6]';
+        this.shinningMethod =
+          '“ส่องแสงของเธอด้วยการส่งต่อความช่วยเหลือของเธอไปยังทุกคนที่เธอสามารถช่วยได้! เริ่มจากแบ่งขนมอร่อยๆ ให้คุณน้าข้างบ้านที่ชอบมาเม้ากับแม่เราดูสิ!  ᕕ( ᐛ )ᕗ ”';
+        break;
+      default:
+        break;
+    }
+  }
+}
```

Comment:
Saggestion: mock as array and user array.find be better

Reply:
can i use .filter() ??

Reply:
Done